smac/scenario/scenario.py

```diff
@@ -190,6 +189,10 @@ def _add_arguments(self):
                           dest='train_insts')
         self.add_argument(name='test_instances', default=[[None]], help=None,
                           dest='test_insts')
+        self.add_argument(name='initial_incumbent', default="DEFAULT",
+                          help=None, dest='initial_incumbent',
+                          callback=lambda arg: str.upper(arg.strip()) if
+                          str.upper(arg.strip()) in ['DEFAULT', 'RANDOM'] else None)
```

Comment:
Hm, although we should have a choice here, a hotfix would be raising an exception in the else statement. Would that make sense?